client/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,23 @@
+interface cardProps {
```

Comment:
I think it might be better to get the `Card` Type and use that in the component? Then you know you have all the right properties of the card and you'll get the `Stat[]` type too.
https://github.com/TeamTopTrumps/top-trumps-game/blob/main/client/src/types/card/card.types.tsx

Reply:
Sure, I will make use of Card and Stat[] types. We were just trying to do something and then ran into conflicts and merge issues. i will look at this later. Thanks!